### PR TITLE
Fixes documentation and changes default save directory of graph.py

### LINKED ISSUE #151: Type of `show_controls` in the example code
Hi @formersbach,
I thinks the type of `show_controls` in the example code of `dynamic_plot()` should be boolean, not string:
```python
model.dynamic_plot(save_dir='example_dir', file_name='nfkb_dynamic.html' show_controls='True', which_controls=['physics', 'layout'])
```

If yes, I have a fix-doc branch and fix this.
Thank you!

### PR DESCRIPTION
**Pull request by formersbach**
Closes #151 & #153

### CODE REVIEW

File: biomass/graph.py [hunk 1]
```diff
@@ -226,12 +226,14 @@ def static_plot(
             raise ValueError(
                 f"gviz_prog must be one of [{', '.join(available_layout)}], got {gviz_prog}."
             )
+        if save_dir == "":
+            save_dir = self.path
```
Comment (himoto) [inline biomass/graph.py:230]: Just a small comment:
Empty string can be detected via `if not`. In this case,
```python
if not save_dir:
    save_dir = self.path
```

File: biomass/graph.py [hunk 2]
```diff
@@ -266,6 +272,8 @@ def dynamic_plot(
             self.to_graph()
         if os.path.splitext(file_name)[1] != ".html":
             file_name = file_name + ".html"
+        if save_dir == "":
+            save_dir = self.path
```
Comment (himoto) [inline biomass/graph.py:276]: Save as above.

**Review by himoto — COMMENTED** [review]
Hi @formersbach, thank you for the improvements!

File: biomass/graph.py (hunk 2 above)
Comment (formersbach) [reply biomass/graph.py]: Changed!

**Comment by himoto** [comment]
Thanks @formersbach!

**Pull request merged**